webportal/webapp/src/environments/environment.prod.ts

```diff
@@ -3,12 +3,13 @@ export const environment = {
   production: true,
   STREAMING_URL: 'https://stream.securedatacommons.com/guacamole/?authToken=',
   REGION: 'us-east-1',
-  USER_POOL_ID: 'us-east-1_Y5JI7ysvY',
-  CLIENT_ID: '7t034p7j0aklgipi5kk8ll5udd',
+  USER_POOL_ID: 'us-east-1_1zLqP2XrI',
+  CLIENT_ID: 'gt6jl7jeqogmelu5tm7ce1vnb',
   IDENTITY_PROVIDER: 'USDOT-ADFS',
-  APP_DOMAIN: 'dev-sdc-dot-webportal',
+  APP_DOMAIN: 'prod-sdc-dot-webportal',
   REDIRECT_URL: 'https://portal.securedatacommons.com/index.html',
-  API_ENDPOINT: 'https://cyfca70occ.execute-api.us-east-1.amazonaws.com/api/',
-  LOGIN_GOV_COGNITO_APP_CLIENT_ID: '34vj4387sna9jk31gudpt3b78i',
+  API_ENDPOINT: 'https://bzzlrrsxa4.execute-api.us-east-1.amazonaws.com/api/',
+  LOGIN_GOV_COGNITO_APP_CLIENT_ID: '3mlku34f58oeloqqc3hpiko6l3',
   LOGIN_GOV_ACCOUNT_LINK_URL: 'https://2l418womp5.execute-api.us-east-1.amazonaws.com',
+  ENVIRONMENT: 'prod',
```

Comment:
Do we need this? We have production: true set already.

Reply:
Oh, are you trying to differentiate between dev and test?

Reply:
Yep. Previously with 2 environments asking if the environment was `prod` was enough. Now we actually have to know.